### DISCUSSION TITLE
Error when running --set-string with helmfile apply , does it supported ?

### DISCUSSION THREAD

**Discussion by swati-delphix** [helmfile/helmfile#1204] [Questions]
As helm chart, we can run --set or --set-string to set values, also we can run --set in helmfile.
In helmfile apply on running below command, its given error as --set-string unknown.

`
helmfile apply --set-string x.ingress.annotations.appgw\\.ingress\\.kubernetes\\.io/override-frontend-port="8442" --set-string identity.env.ENV_PORT="8442"
`

**Comment by swati-delphix** [comment]
Any update ?

**Reply by yxxhero** [reply]
@swati-delphix PR is welcome.
